XenAdmin/Wizards/BugToolWizardFiles/BugToolPageDestination.cs

```diff
@@ -206,7 +206,7 @@ private bool CheckPathValid(out string error)
             if (String.IsNullOrEmpty(name))
                 return false;
 
-            if (!PathValidator.IsFileNameValid(name, out string invalidNameMsg))
+            if (!XenModel.PathValidator.IsFileNameValid(name, out string invalidNameMsg))

```

Comment:
Here and in the following 4 files: it makes the code more readable if XenModel moves to a `using` directive.